code/__DEFINES/is_helpers.dm

```diff
@@ -158,6 +158,10 @@ GLOBAL_LIST_INIT(turfs_without_ground, typecacheof(list(
 
 #define isaicamera(A) (istype(A, /mob/camera/ai_eye))
 
+//SKYRAT EDIT ADD START
```

Comment:
Skyrat "is" helpers go in `code\__DEFINES\~skyrat_defines\is_helpers.dm`

Reply:
oops